content/about/privacy.md

```diff
@@ -0,0 +1,96 @@
+
+- Voluntarily answer questionnaires and surveys: From time to time we may send you questionnaires or surveys for collecting demographic information or assessing your interests and needs, to help us to improve our services. Any such questionnaires or surveys will be entirely voluntary. The purpose and intended use of the information being collected will be explained in the survey itself.
+
+### Information Automatically Collected From You
+
+We also collect certain information automatically. Like many websites, the InnerSource Commons  website uses “cookies” and we obtain certain types of information when your web browser accesses our site. On the InnerSource Commons website Cookies are used for the following purposes and using the following services:
+ 
+Analytics
+
+- Google Analytics - Personal Data: Cookies and Usage Data. We track number of visitors per page on our website.
+
+Displaying content from external platforms
+
+- Twitter - Personal Data: Usage Data and various types of Data as specified in the [privacy policy](https://twitter.com/en/privacy) of the service
+- Youtube - Personal Data: Usage Data and various types of Data as specified in the [privacy policy](https://policies.google.com/privacy) of the service
+
+Forms
+
+- MailChimp (emailing list) - Personal Data: Cookies and various types of Data as specified in the [privacy policy](https://mailchimp.com/legal/privacy/) of the service
+
+You can set or amend your web browser controls to accept or refuse cookies. As the means by which you can refuse cookies through your web browser controls vary from browser-to-browser, you should visit your browser's help menu for more information.
+
+## How and When Do We Share Your Information?
+
+InnerSource Commons is not in the business of selling your information. As set out below, we only share information on a limited basis in order to enable us to offer our services. We do not otherwise make your data available to third parties.  
+
+### Service Providers
+
+We employ other organizations and service providers to perform certain functions on our behalf. These third parties have only limited access to your information, may use your information only to perform these tasks on our behalf, and are obligated to InnerSource Commons not to disclose or use your information for other purposes. 
+
+Our use of third-party service providers includes:
+- GitHub - to host and contribute to projects
+- Mailchimp - store an email list for marketing purposes
+- Slack - for community communication
+- Eventbrite???
+
```

Comment:
Zoom? Jit.si?

Reply:
I am guessing we use Zoom for conferences and community meetings. 
What do we use Jit.si for?

Reply:
@Katiegilroyy what is the platform we are using for events? The equivalent to Eventbrite?

Reply:
@clcoffey it's [eventyay](https://eventyay.com/e/3dbaaa50)

Reply:
> What do we use Jit.si for?

Some working groups use that for their group calls like we use Zoom.

Reply:
Thanks for that I updated it in the last commit.